api/v1alpha1/dexserver_types.go

```diff
@@ -194,6 +198,9 @@ type DexServerStatus struct {
 	Message string `json:"message,omitempty"`
```

Comment:
Shall we ditch `State` and `Message` from `DexServerStatus`? I don't believe we're using them now that we're just using Conditions.

Reply:
Not sure about removing so leaving for now.